barcamp-theme/templates/base.html

```diff
@@ -61,7 +61,11 @@
 
 		    <p class="lead">Beaver BarCamp is an informal conference where everyone is encouraged to participate and the sessions are not predetermined. <a href="/pages/about.html"><em>Learn more</em></a></p>-->
 		    <h5>Event Sponsors</h5>
-        <p>Want to sponsor the event? Please contact info < at > beaverbarcamp.org</p>
+            <img class='sponsor' src='/theme/img/github.png' />
```

Comment:
Remove all of these leading ``/``. When I build the site, it tries accessing from ``/theme`` which is not right. Leaving it out ensures its relative. Please make the change to the other images.